pom.xml

```diff
@@ -31,6 +31,14 @@
             <artifactId>selenium-java</artifactId>
             <version>4.7.2</version>
         </dependency>
+        
```

Comment:
TestNG dependency is already defined in pom.xml file so please remove all of these lines for testng dependency added by you. No need to added multiple times and duplicate the code.

Reply:
am comentat sa- mi spui daca sigur sunt acelea care trebuie sterse